### PR TITLE
Need medic! pipeline

### LINKED ISSUE #566: Rendering Pipeline Checkup
Going through the Rendering Pipeline to make sure it's healthy and well.

### PR DESCRIPTION
**Pull request by Ittis**
Fixed nSight and ErrorVideoDebug.hpp. you can now ask openGL for Errors.
Fixes #566

### CODE REVIEW

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/Video/ErrorVideoDebug.hpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+
+#include <Utility/Log.hpp>
+#include <GL/glew.h>
+
+#define ERROR_CHECK_VIDEO_START {while(glGetError() != GL_NO_ERROR){Log()<<"GL Error\n";}}
+#define ERROR_CHECK_VIDEO_END { GLenum __err(glGetError()); while (__err != GL_NO_ERROR) { Log() << "GL error: " << (const char*)gluErrorString(__err) << "\n"; __err = glGetError(); assert(false); } }
```
Comment (Chainsawkitten) [inline src/Video/ErrorVideoDebug.hpp]: Missing newline at end of file.

File: src/Video/Buffer/StorageBuffer.cpp [hunk 2]
```diff
@@ -5,8 +5,7 @@
 
 
 #include <Utility/Log.hpp>
-#define ERROR_CHECK_VIDEO { GLenum __err(glGetError()); while (__err != GL_NO_ERROR) { Log() << "GL error: " << (const char*)gluErrorString(__err) << "\n"; __err = glGetError(); assert(false); } }
-
+#include<ErrorVideoDebug.hpp>
```
Comment (Chainsawkitten) [inline src/Video/Buffer/StorageBuffer.cpp]: Space between include and <

File: src/Video/ErrorVideoDebug.hpp (hunk 1 above)
Comment (Ittis) [reply src/Video/ErrorVideoDebug.hpp]: Nope

**Pull request merged**